presto-main/src/test/java/com/facebook/presto/execution/TestMemoryRevokingScheduler.java

```diff
@@ -113,71 +113,70 @@ public void tearDown()
     public void testScheduleMemoryRevoking()
             throws Exception
     {
-        // todo test for different threshold/target
         SqlTask sqlTask1 = newSqlTask();
         SqlTask sqlTask2 = newSqlTask();
 
-        TaskContext taskContext1 = sqlTask1.getQueryContext().addTaskContext(new TaskStateMachine(new TaskId("q1", 1, 1), executor), session, false, false);
-        PipelineContext pipelineContext11 = taskContext1.addPipelineContext(0, false, false);
-        DriverContext driverContext111 = pipelineContext11.addDriverContext();
-        OperatorContext operatorContext1 = driverContext111.addOperatorContext(1, new PlanNodeId("na"), "na");
-        OperatorContext operatorContext2 = driverContext111.addOperatorContext(2, new PlanNodeId("na"), "na");
-        DriverContext driverContext112 = pipelineContext11.addDriverContext();
-        OperatorContext operatorContext3 = driverContext112.addOperatorContext(3, new PlanNodeId("na"), "na");
+        TaskContext q1 = sqlTask1.getQueryContext().addTaskContext(new TaskStateMachine(new TaskId("q1", 1, 1), executor), session, false, false);
+        PipelineContext q1_p1 = q1.addPipelineContext(0, false, false);
+        DriverContext q1_p1_d1 = q1_p1.addDriverContext();
+        OperatorContext q1_p1_d1_o1 = q1_p1_d1.addOperatorContext(1, new PlanNodeId("na"), "na");
+        OperatorContext q1_p1_d1_o2 = q1_p1_d1.addOperatorContext(2, new PlanNodeId("na"), "na");
```

Comment:
I will rollback this change because those variable names are rejected by checkstyle.